rag_experiment_accelerator/config/index_config.py

```diff
@@ -38,14 +41,17 @@ def index_name(self) -> str:
         """
         index_name = (
             f"{self.index_name_prefix}"
-            f"_{str(self.chunk_size)}"
-            f"_{str(self.overlap)}"
-            f"_{str(self.embedding_model.name.lower())}"
-            f"_{str(self.ef_construction)}"
-            f"_{str(self.ef_search)}"
+            f"_cs-{str(self.chunk_size)}"
```

Comment:
I feel that this is becoming a bit unwieldy. Do we want all of these parameters in the index? I think it isn't possible anyway to capture _everything_ about the parameters in the index name, as there can be other changes (code, configuration...). Could we perhaps save this metadata elsewhere (not as part of this PR)?

Reply:
this is important because if we hold several configurations they will override the same index...